internal/util/target.go

```diff
@@ -95,7 +33,7 @@ func ShootNamesForTarget(ctx context.Context, manager target.Manager, t target.T
 		return nil, fmt.Errorf("failed to create Kubernetes client for garden cluster %q: %w", t.GardenName(), err)
 	}
 
-	shoots, err := ShootsForTarget(ctx, gardenClient, t)
+	shoots, err := gardenClient.ListShoots(ctx, t.WithShootName("").AsListOptions()...)
```

Comment:
`// target must either target a project or a seed (both including a garden).`
can be removed

Reply:
:marked